Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -3595,7 +3604,787 @@ script:
+      description: the number of files that were quarantined during the scan.
+      type: Number
+    - contextPath: CrowdStrike.ODSScanHost.filecount.skipped
+      description: the number of files that were skipped during the scan.
+      type: Number
+    - contextPath: CrowdStrike.ODSScanHost.status
+      description: the status of the scan. (e.g., "completed", "pending", "cancelled", "running", or "failed")
+      type: String
+    - contextPath: CrowdStrike.ODSScanHost.severity
+      description: A severity score assigned to the scan, ranging from 0 to 100
+      type: Number
+    - contextPath: CrowdStrike.ODSScanHost.started_on
+      description: the date and time when the scan was started.
+      type: Date
+    - contextPath: CrowdStrike.ODSScanHost.completed_on
+      description: the date and time when the scan was completed.
+      type: Date
+    - contextPath: CrowdStrike.ODSScanHost.last_updated
+      description: the date and time when the scan event was last updated.
+      type: Date
+  - arguments:
+    - description: Valid CS-Falcon-FQL filter to query with.
+      name: filter
+      required: false
+    - description: Comma-separated list of malicious file IDs to retrieve details about. If set will override all other arguments.
+      isArray: true
+      name: file_ids
+      required: false
+    - description: Comma-separated list of host IDs to filter by.
+      isArray: true
+      name: host_ids
+      required: false
+    - description: Comma-separated list of scan IDs to filter by.
+      isArray: true
+      name: scan_ids
+      required: false
+    - description: Comma-separated list of file paths to filter by.
+      isArray: true
+      name: file_paths
+      required: false
+    - description: Comma-separated list of file names to filter by.
+      isArray: true
+      name: file_names
+      required: false
+    - description: Comma-separated list of hashes to filter by.
+      isArray: true
+      name: hash
+      required: false
+    - description: Starting index of overall result set from which to return ids.
+      name: offset
+      required: false
+    - description: Maximum number of resources to return.
+      name: limit
+      required: false
+    description: Retrieve ODS malicious file details.
+    name: cs-falcon-ods-query-malicious-files
+    outputs:
+    - contextPath: CrowdStrike.ODSMaliciousFile.id
+      description: A unique identifier of the detection event.
+      type: String
+    - contextPath: CrowdStrike.ODSMaliciousFile.cid
+      description: A unique identifier for the client that triggered the detection event.
+      type: String
+    - contextPath: CrowdStrike.ODSMaliciousFile.scan_id
+      description: A unique identifier for the scan that triggered the detection event.
+      type: String
+    - contextPath: CrowdStrike.ODSMaliciousFile.host_id
+      description: A unique identifier for the scan that detected the file on the host.
```

Comment:
This description is the same as the next one.  Is that correct?

Reply:
fixed